Examples/torch/quantization/autoquant.ipynb

```diff
@@ -10,39 +10,59 @@
    "source": [
     "# AutoQuant\n",
     "\n",
-    "This notebook shows a working code example of how to use AIMET AutoQuant feature.\n",
+    "This notebook contains an example of how to use AIMET AutoQuant feature.\n",
     "\n",
-    "AIMET offers a suite of neural network post-training quantization (PTQ) techniques that can be applied in succession. However, the process of finding the right combination and sequence of techniques to apply is time-consuming and requires careful analysis, which can be challenging especially for non-expert users. We instead recommend AutoQuant to save time and effort.\n",
+    "AIMET offers a suite of neural network post-training quantization (PTQ) techniques that can be applied in succession. However, finding the right sequence of techniques to apply is time-consuming and can be challenging for non-expert users. We instead recommend AutoQuant to save time and effort.\n",
     "\n",
-    "AutoQuant is an API that applies various PTQ techniques in AIMET automatically based on analyzing the model and best-known heuristics. In AutoQuant, users specify the amount of tolerable accuracy drop, and AutoQuant will apply PTQ techniques cumulatively until the target accuracy is satisfied.\n",
+    "AutoQuant is an API that analyzes the model and automatically applies various PTQ techniques based on best-practices heuristics. You specify a tolerable accuracy drop, and AutoQuant applies PTQ techniques cumulatively until the target accuracy is satisfied.\n",
     "\n",
+    "## Overall flow\n",
+    "\n",
+    "This example performs the following steps:\n",
     "\n",
-    "#### Overall flow\n",
-    "This notebook covers the following\n",
     "1. Define constants and helper functions\n",
     "2. Load a pretrained FP32 model\n",
     "3. Run AutoQuant\n",
     "\n",
-    "#### What this notebook is not\n",
-    "This notebook is not designed to show state-of-the-art AutoQuant results. For example, it uses a relatively quantization-friendly model like Resnet18. Also, some optimization parameters are deliberately chosen to have the notebook execute more quickly.\n"
+    "<div class=\"alert alert-info\">\n",
+    "\n",
+    "Note\n",
+    "\n",
+    "This notebook does not show state-of-the-art QAT results. For example, it uses a relatively quantization-friendly model (Resnet18). Also, some optimization parameters like number of fine-tuning epochs are chosen to improve execution speed in the notebook.\n",
```

Comment:
Copy - paste error, it shouldn't say state-of-the-art QAT results in autoquant.ipynb